docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
 
-To determine the operating systems you should continue to support, and the importance of mobile traffic from iOS devices, 
-create a donut chart that displays the top operating systems that your visitors used to access your website within the last 90 days.
+.. Drag the bottom corner of the panel until the metric takes up one quarter of the screen
+width. The row for the metric will have 4 items on it later.
 
-. Open *Lens*, then set the <<set-time-filter,time filter>> to *Last 90 days*.
+.. The metric visualization has its own label, so you do not need to add a panel title.
 
-. From the *Chart Type* dropdown, select *Donut*.
+. Click *Save* on the dashboard menu
 
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
+. Enter the title "Logs dashboard"
 
-.. Drag and drop *clientip* to the *Size by* field.
-
-.. Drag and drop *machine.os.keyword* to the *Slice by* field.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_2_1_1.png[Donut chart with clientip and machine.os.keyword fields]
-
-. Change the color palette. 
-
-.. In the editor, click *Top values of machine.os.keyword*. 
-
-.. From the *Color palette* dropdown, select *Compatibility*.
-
-.. Click *Close*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_2_1.png[Donut chart with open config panel]
+. Check the *Store time with dashboard* box, then click *Save*
 
-. Click *Save and return*.
+. After the dashboard refreshes, click *Edit* again.
 
 [discrete]
 [[mixed-multiaxis]]
-=== View the average of bytes transfer per day
+=== View a metric over time
 
-To prevent potential server failures, and optimize the cost of website maintenance, create an area chart that displays the average of bytes transfer. To compare 
-the data to the number of visitors to your website, add a line chart layer.  
+If your index pattern has a default time field, *Lens* will add that field automatically
+when you drag a numeric field onto the workspace panel. This is a helpful shortcut.
+If your data has multiple time fields you can drag to replace the previous time field.
 
-. Open *Lens*.
+For this example, you want to visualize the `bytes` field over time, but you may not have
+chosen a specific function yet:
 
-. From the *Available fields* list, drag and drop *bytes* to the visualization builder.
+. Drag `bytes` from the field list to the workspace panel to have *Lens* automatically
+create a chart. It will create a bar chart using the *median of bytes* as the metric.
+
+. *Lens* chooses a date interval automatically. To zoom in on the data you want to view,
+click and drag your cursor across the bars. 
 
-. To zoom in on the data you want to view, click and drag your cursor across the bars. 
-+
 [role="screenshot"]
 image::images/lens_end_to_end_3_1_1.gif[Zoom in on the data]
 
-. Change the *timestamp* interval.
+Bar charts are a good default chart, but for this metric a line chart is more appropriate.
```

Comment:
I'm unsure about what `good default` means? 

What is `metric` referring to? Do you mean `function`? This is confusing because we have a Metric visualization.

Why is a line chart more appropriate? What does the line chart show that a bar chart cannot?

Reply:
Line charts are the best choice for continuous metrics as it highlights the change from point to point better than a bar chart. I can clarify this.

I think the right word here is metric. We use the word metric in Lens, Visualize, and Elasticsearch to refer to the basic aggregations like Count/Median/Avg, but not to refer to bucket aggregations like Top Values/Date Histogram. Function is a word we use in the Lens UI as well, but metric is a more specific word than function.